cobl2/__init__.py

```diff
@@ -66,4 +67,12 @@ def main(global_config, **settings):
     config.add_route('test', '/test')
     config.registry.registerUtility(link_attrs, ILinkAttrs)
     config.registry.registerUtility(CoblMapMarker(), IMapMarker)
+    menuitems = [
+            'languages',
+            'parameters',
+            ('cognatesets', dict(label='Cognate Sets')),
```

Comment:
Is there a reason to not use the translations solution here? Presumably, "Cognatesets" appears more often in the user interface of the app?

Reply:
how can be queried the translated name here?